lib/tests/test_shape_utils.py

```diff
@@ -140,3 +146,47 @@ def test_drop_parts_of_geoms(self, capsys, europe_gdf):
             europe_gdf[europe_gdf.name == 'Highlands and Islands'].area
             > gdf[gdf.name == 'Highlands and Islands'].area
         ).all()
+
+class TestPolygonsFromPoints:
+
+    @pytest.fixture
+    def points(self):
+        points = {  # Latitiude, Longitude
+            'athens': [37.9838, 23.7275],
+            'davos': [46.8027, 9.8360],
+            'stockholm': [59.3293, 18.0686],
+            'wurzburg': [49.7913, 9.9534]
+        }
+        areas = {
+            'athens': 1,
+            'davos': 5,
+            'stockholm': 100,
+            'wurzburg': 500
+        }
+        point_gdf = gpd.GeoDataFrame(
+            # x = longitude, y = latitude
+            geometry=gpd.points_from_xy([i[1] for i in points.values()], [i[0] for i in points.values()]),
+            index=points.keys(),
+            crs = WGS84
+        )
+        point_gdf["REP_AREA"] = pd.Series(areas)
+        return point_gdf
+
+    @pytest.fixture
+    def estimated_polygons(self, points):
+        return estimate_polygons_from_points(points, "REP_AREA")
+
+
+    @pytest.mark.parametrize(("area", "radius"), [(np.pi, 1000), (1, 564.19)])
+    def test_radius_meter(self, area, radius):
+            calculated_radius = _radius_meter(area)
+            assert np.isclose(calculated_radius, radius)
+
+    def test_estimate_polygons_area(self, points, estimated_polygons):
+        assert estimated_polygons.crs == points.crs
```

Comment:
Usually there's one `assert` per test. This `assert` here seems unnecessary: you are testing this in the test below!